### PR TITLE
alarm clock

### PR DESCRIPTION
**Pull request by FaisalTayyab**
I implemented Alarm Clock using Python. I also added a read me file explaining the code. Hoping to get the PR merged and getting the Hacktoberfest and hacktoberfest- accepted badge

### CODE REVIEW

File: B/battleship game/readme.md
Comment (FaisalTayyab) [inline B/battleship game/readme.md:1]: I implemented battleship game using Python. I also added a read me file explaining the code. Hoping to get the PR merged and getting the Hacktoberfest and hacktoberfest- accepted badge

**Review by joshiadarshx — COMMENTED** [review]
Code looks clean and readable .I think adding comments can make it more better and readable

**Comment by nhuzaa** [comment]
Looks simple and clean code

**Review by joshiadarshx — COMMENTED** [review]
Clean code good one

**Pull request merged**